### DISCUSSION TITLE
Not getting weather observation data for PHNL and PHJR stations

### DISCUSSION THREAD

**Discussion by sthapa123** [weather-gov/api#220] [Q&A]
I was trying to get weather observation data for PHNL and PHJR stations. I'm only getting null values for  attributes like temperature, dewpoint, relativeHumidity, etc. Is there any alternative to get weather observation data for these stations?

**Comment by hluker** [comment]
Definitely looks like a bug for the parsing of these. Here's another source you can use (not as complete with weather conditions and such though) http://mesonet.agron.iastate.edu/api/1/currents.json?station=PHNL
